VimeoNetworking/Sources/KeychainStore.swift

```diff
@@ -109,29 +109,29 @@ final class KeychainStore
      */
     func deleteDataForKey(key: String) throws
     {
-        let query = self.queryForKey(key)
+        let query = self.queryForKey(key: key)
         
-        let status = SecItemDelete(query)
+        let status = SecItemDelete(query as CFDictionary)
         
         if status != errSecSuccess && status != errSecItemNotFound
         {
-            throw self.errorForStatus(status)
+            throw self.errorForStatus(status: status)
         }
     }
     
     // MARK: - 
     
-    private func queryForKey(key: String) -> [String: AnyObject]
+    private func queryForKey(key: String) -> [String: Any]
```

Comment:
I think in this case it might be worth casting to the CFDictionary as necessary unless there's a good reason not to. Being able to use subscripting to set values is a pretty nice improvement over the regular CFDictionary APIs. 

How strongly do you feel about this one?